docs/content/feature/access-control.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Access Control"
+since: "1.5.8"
+---
+
+fabio supports basic ip centric access control per route.  You may
+specify one of `allow` or `deny` options per route to control access.
+Currently only source ip control is available.
```

Comment:
Pls add an ipv6 example and explain how the source ip is determined. Also add a comment that `/32` and `/128` is added automatically.

Reply:
done!